docs/changelog/1303.bugfix.rst

```diff
@@ -0,0 +1 @@
+Allow virtualenv creation with older pip not having ``config`` command.
```

Comment:
Can you here note that in this case configuration related features (such as pip cert setting) will not work?

Reply:
Updated changelog entry but if you have better formulation I'll gladly use it.